minisite.theme.inc

```diff
@@ -101,68 +102,17 @@ function template_preprocess_minisite_link(&$variables) {
   }
 
   // Classes to add to the file field for icons.
-  $classes = array(
+  $classes = [
     'file',
     // Add a specific class for each and every mime type.
-    'file--mime-' . strtr($mime_type, array('/' => '-', '.' => '-')),
+    'file--mime-' . strtr($mime_type, ['/' => '-', '.' => '-']),
     // Add a more general class for groups of well known MIME types.
     'file--' . file_icon_class($mime_type),
-  );
+  ];
 
   // Set file classes to the options array.
   $variables['attributes'] = new Attribute($variables['attributes']);
   $variables['attributes']->addClass($classes);
 
-  $variables['link'] = \Drupal::l($link_text, Url::fromUri($url, $options));
-}
-
-/**
- * Implements hook_theme_registry_alter().
- */
-function minisite_theme_registry_alter(&$theme_registry) {
```

Comment:
this was adding a theme for a minisite page, served through alias. removed for now.